### ISSUE TITLE
Did you test this @20v

### ISSUE THREAD

**Issue by abrecht** [t3st-account/rioby2pd#1]
Hi, I would be curious how this performs when 20v is requested by the consuming device. Were you able to do any testing? 
What is the typical output voltage of the 18v ryobi battery?
Is there any protection against deep discharge?
Thanks for publishing! 👍

**Comment by t3st-account** [comment]
Yes I tested at 20v, my thinkpad charged until the battery went to 1 out of 4 leds.
 a full genuine Rioby battery puts out 20.5v open circuit.
 Testing with a Power delivery trigger board i get exactly 20.0v open circuit.
With a ts100 slodering iron @45w (reported by the iron) the voltage drops to 19.5v.

Deep discarge protection is not provided by the PD module, but genuine (and my fake) batteries have a BMS with undervoltage protection built in, as far as I know most Rioby tools are pretty dumb and rely on the battery to save itself.

Hope that helps.

**Comment by t3st-account** [comment]
I'll do more tests once my battery is discharged a bit, and report back

**Comment by t3st-account** [comment]
Ok did some more testing.
At 18.7V battery voltage the PD module puts out 18.4V @43W
